core/controllers/feedback_test.py

```diff
@@ -554,3 +556,216 @@ def test_feedback_threads_with_suggestions(self):
         }
         self.assertDictContainsSubset(
             expected_suggestion_dict, response['suggestion'])
+
+    def test_get_feedback_threads_with_no_text_and_no_updated_status_raises_400(
```

Comment:
This name doesn't seem to correspond with what the test is doing.... You seem to be checking the 400 error on a post request and not a get request (as the name suggests). Can you explain what exactly this tests and why this test is necessary? (I don't seem to understand the flow).

Reply:
Oops should have been post. Changed it!